### PR TITLE
Setup test suite

### PR DESCRIPTION
**Pull request by tysongach**
* Setup test suite with rspec and css parser
* Add test function for modular-scale

### CODE REVIEW

**Comment by joshuaogle** [comment]
Aside from those things that Hound caught, this is looking great. Good job, guys! Really looking forward to having this in.

Those old cucumber tests sure are noisy. I completely missed that it was also running the new ones when you run `rake`. Are we planning on updating those?

**Comment by joshuaogle** [comment]
At what point do we add documentation about tests? Do we want to sneak a few more in first?

File: bourbon.gemspec [hunk 1]
```diff
@@ -28,5 +28,7 @@ meaning they should be as close to the original CSS syntax as possible.
   s.add_dependency('thor')
 
   s.add_development_dependency('aruba', '~> 0.4')
-  s.add_development_dependency('rake')
+  s.add_development_dependency('rake') 
+  s.add_development_dependency('rspec', '~> 2.14.1')
```
Comment (teoljungberg) [inline bourbon.gemspec]: why is `rspec` locked?

File: Rakefile [hunk 2]
```diff
@@ -2,8 +2,10 @@
 require 'rubygems'
 require 'bundler'
 require 'cucumber/rake/task'
+require 'rspec/core/rake_task'
```
Comment (teoljungberg) [inline Rakefile]: I don't know why hound missed this.. replace with double quotes

**Comment by donokuda** [comment]
Looking good here to me :+1:

**Pull request merged**